CMakeLists.txt

```diff
@@ -92,12 +92,14 @@ if(NOT DAEMON_EXTERNAL_APP)
 
     if (CMAKE_SYSTEM_NAME STREQUAL CMAKE_HOST_SYSTEM_NAME)
         # can be loaded by daemon with vm.[sc]game.type 0 or 1
-	option(BUILD_GAME_NACL "Build the NaCl gamelogic modules, required to host mods." OFF)
-	option(BUILD_GAME_NACL_NEXE "Build the NaCl \"nexe\" files from the \"pexe\" files." ON)
-	mark_as_advanced(BUILD_GAME_NACL_NEXE)
+        option(BUILD_GAME_NACL "Build the NaCl gamelogic modules, required to host mods." OFF)
```

Comment:
Why are these lines diffs? It looks exactly the same.

Reply:
there was just random tab/space mix that were annoying me

Reply:
The request to not include superfluous changes in the big commit applies then

Reply:
Moved to dedicated bikeshedding commit.